### PR TITLE
Cleaning up some stuff (flask.ext, missing requirement, etc)

### PR DESCRIPTION
**Pull request by ThiefMaster**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,5 +4,6 @@ flask-assets
 flask-script
 flask-sqlalchemy
 flask-wtf
+psycopg2
```
Comment (pferreir) [inline requirements.txt]: Who says the guy will be using postgres?

Comment (ThiefMaster) [inline requirements.txt]: The example config does ;)

Comment (pferreir) [inline requirements.txt]: It's just the example config. It's not mandatory. `requirements.txt` is :wink:

**Comment by pferreir** [comment]
Thanks!

**Pull request merged**